src/dropdown/Dropdown.js

```diff
@@ -5,6 +5,8 @@ import cx from 'classnames';
 import { props, t } from '../utils';
 import { warn } from '../utils/log';
 
+import './dropdown.scss';
```

Comment:
This is a breaking change we don't want: every old project built without webpack is going to break if you add this import.
Please remove